asciidoctorj-core/src/main/java/org/asciidoctor/extension/Processor.java

```diff
@@ -40,42 +40,59 @@
      * </verbatim>
      * </pre>
      * </p>
+     *
+     * @deprecated Use the annotation {@link ContentModel} instead to describe this Processor
      */
+    @Deprecated
     public static final String CONTENT_MODEL = "content_model";
```

Comment:
I don't think there's an equivalent for this constant, it there? Perhaps we can move it to ContentModel.KEY? or something.